src/server/app/admin/controllers/users/users.authentication.server.controller.js

```diff
@@ -83,7 +83,7 @@ function signupAlert(user, req, res) {
 		name: user.name,
 		username: user.username,
 		appName: config.app.title,
-		url: `${config.app.baseUrl}/#/admin/users`
```

Comment:
Does/should baseUrl now include the `/#/` suffix? Looks like it's referenced in two places in this PR.